.github/workflows/milestoneclose.yml

```diff
@@ -0,0 +1,82 @@
+name: Closed Milestones
+
+on:
+  milestone:
+    types: [closed]
+
+env:
+  GITHUB_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+  REPO: kbcli
+  milestone: 8
+  ORGANIZATION: apecloud
+  PROJECT_NUMBER: 2
```

Comment:
First create the project with kbcli, and then set it to the number of the project.